qlib/CsvUtil.qm

```diff
@@ -499,18 +661,26 @@ public namespace CsvUtil {
 
                         break;
                     }
+                    case "resolve_type": {
+                        if (i.value.typeCode() != NT_CLOSURE) {
```

Comment:
please use:
`is (!i.value.callp())` instead here  (http://www.qoretechnologies.com/manual/qorus/latest/qore/lang/html/class_qore_1_1zzz8valuezzz9.html#a3bd7145235f0f4160b29e9ff1a925f4f), because we should treat call references and closures identically.